pkg/session/session.go

```diff
@@ -213,8 +214,23 @@ func clearCache(sessionKey string) {
 }
 
 // newManager creates a Manager that encompasses the REST Client for the VSphere tagging API
-func newManager(ctx context.Context, client *vim25.Client, user *url.Userinfo) (*tags.Manager, error) {
+func newManager(ctx context.Context, logger logr.Logger, sessionKey string, client *vim25.Client, user *url.Userinfo, feature Feature) (*tags.Manager, error) {
```

Comment:
Thanks Deepak. Shall we open PR to upstream and have it reviewed there before merge?

Reply:
does this functionality exist in our v1a3 branch?

Reply:
we don't need it in v1alpha3 as this tagmanager is initialized for vspherefailuredomain

Reply:
> Thanks Deepak. Shall we open PR to upstream and have it reviewed there before merge?

Ack